docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,444 @@
+
+- Add support for Kubernetes 1.24.8. Those Kubernetes patch releases fix CVE-2022-3162 and CVE-2022-3294, both in kube-apiserver. We strongly recommend upgrading to those Kubernetes patch releases as soon as possible ([#11340](https://github.com/kubermatic/kubermatic/pull/11340))
+    - [CVE-2022-3162: Unauthorized read of Custom Resources](https://groups.google.com/g/kubernetes-announce/c/oR2PUBiODNA/m/tShPgvpUDQAJ)
+    - [CVE-2022-3294: Node address isn't always verified when proxying](https://groups.google.com/g/kubernetes-announce/c/eR0ghAXy2H8/m/sCuQQZlVDQAJ)
+- Add support for Kubernetes v1.24.9 ([#11553](https://github.com/kubermatic/kubermatic/pull/11553))
+- Add support for Kubernetes 1.25 ([#11049](https://github.com/kubermatic/kubermatic/pull/11049))
+- Add support for Kubernetes 1.26 ([#11621](https://github.com/kubermatic/kubermatic/pull/11621))
+- Support for Kubernetes 1.22 and 1.23 user clusters has been removed; User clusters remaining on 1.23 will be automatically upgraded with this KKP version ([#11286](https://github.com/kubermatic/kubermatic/pull/11286), [#11767](https://github.com/kubermatic/kubermatic/pull/11767))
+- Allow Kubernetes version upgrade for clusters with non-amd64 nodes & Canal CNI and IPVS for all Kubernetes versions ([#11765](https://github.com/kubermatic/kubermatic/pull/11765))
+
+#### Supported Versions
+
+- 1.24.8
+- 1.24.9
+- 1.25.4 
+- 1.25.5
+- 1.26.0
+
+### Highlights
+
+#### KubeVirt
+
+KubeVirt cloud provider support is leaving the "technical preview" phase and is now considered GA.
+
+- Fix to ensure that we do not raise an error when reconciling the namespace in the infrastructure KubeVirt cluster, until we get the value of the namespace to create, avoiding transient errors ([#10849](https://github.com/kubermatic/kubermatic/pull/10849))
+- Bugfix for KubeVirt infra CSI token creation due to auto-creation disabled in k8s 1.24 ([#10908](https://github.com/kubermatic/kubermatic/pull/10908))
+- Introduction of KubeVirt default instance types and instance preferences that will replace virtual machine presets ([#11025](https://github.com/kubermatic/kubermatic/pull/11025))
+- Fix wrong quota filtering when `VirtualMachineInstancePreset.spec.cpu` has no quantity but only other fields ([#11046](https://github.com/kubermatic/kubermatic/pull/11046))
+- Add support for ToplogySpreadConstraint for Kubevirt VM ([#11114](https://github.com/kubermatic/kubermatic/pull/11114))
+- Add support for instancetype/preference ([#11182](https://github.com/kubermatic/kubermatic/pull/11182))
+- Update KubeVirt CCM to v0.4.0 and kubevirt.io/api to v0.58.0 ([#11249](https://github.com/kubermatic/kubermatic/pull/11249))
+- Add Images option in KubeVirt DataCenter which supports download and caching of VM images from HTTP source ([#11302](https://github.com/kubermatic/kubermatic/pull/11302))
+- Add DataVolumeSize field to seed and ImageCloningEnabled to cluster ([#11651](https://github.com/kubermatic/kubermatic/pull/11651))
```

Comment:
Please remove this as the custom images were not delivered.

Reply:
Removed both lines, thanks for the heads up.